active_directory/tox.ini

```diff
@@ -14,4 +14,4 @@ deps =
     -rrequirements-dev.txt
 commands =
     pip install -r requirements.in
-    pytest -v {posargs}
+    pytest -v --junit-xml=junit/test-{envname}.xml {posargs}
```

Comment:
should it be `.junit` so it does not look like a junit integration?

Reply:
@hithwen The `junit` folder is inside the integration folder e.g. `etcd/junit/test-py27.xml` 

But it's a good idea anyway to add `.` so fixed in my last commit :)